core/templates/partial-approach.html

```diff
@@ -125,5 +127,9 @@ <h5>Optional – Submit your algorithm in the form of a Docker container</h5>
             document.getElementById('file-name').innerHTML = file.files[0].name;
         }
     };
+
+    $(function() {
```

Comment:
@sgratzl do you see any value in attempting to do this work by customizing the widget in `forms.py` like so: https://docs.djangoproject.com/en/2.2/ref/forms/widgets/#styling-widget-instances.

Reply:
it would have to be a nested class, since it has to be directly applied to the button element. Usual styling just deals with the root element. Moreover, the button itself it generated by the javascript code not the django widget which makes it more difficult.